### ISSUE TITLE
Limit the number of friends that can be added to a region

### ISSUE THREAD

**Issue by NLthijs48** [NLthijs48/AreaShop#54]
Limits per region (setting in default.yml) and possibly permissions (maybe multiplier permissions?).

*Suggested by St.Schmitty*

**Comment by ElloElloEllo** [comment]
I can't remember anymore why I suggested both. I assume it was due to the different ways we use Areashop. For one as storage unit and newcomer flats (1-2 players), where sharing is rather uncommon, as management system for our mall (<3p), server owned flats for rental.

This could be handled the same way as with renting/buying limitGroups (member/vip).
http://dev.bukkit.org/bukkit-plugins/regionbuyandrent/pages/advanced-limit-groups-setup-example/

I guess I didn't make the connection at first and thats why I went for regions and permission.

The case where I came up with the suggestion, was our factory setup where CEOs have different tiers which allows them to hire a higher number of workers. This is already managed by our own plugin, but it might be useful for other scenarios too.

**Comment by NLthijs48** [comment]
Thanks for your input about the feature. I'm not sure if I want to implement a system like the limitGroups has, because that is quite complicated (code-wise and to use by plugin users). I also think such a system would be a bit overkill for configuring how many players can be added as friend for a certain region. That is why I suggested making a setting in `default.yml` and maybe some permissions to give out multipliers.

**Comment by ElloElloEllo** [comment]
Yes, it is not neccesary. I plan to create a workaround with Denizen as script environment,

**Comment by NLthijs48** [comment]
I propose to implement events that developers can hook into for adding/removing friends, and not implementing settings for it.

**Comment by NLthijs48** [comment]
Events have been implemented, but a setting would still be a good addition.
